docs/DeveloperGuide.md

```diff
@@ -921,36 +921,242 @@ testers are expected to do more *exploratory* testing.
 
    1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+2. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+3. Exiting the program
+
+    1. Enter `exit` command in command input box or click on X button.<br>
+       Expected: The application closes
+
+### Adding a stall
+
+1. Adding a stall
+
+    1. Prerequisites: None
+
+    2. Test case: `sadd n/Ah Kim Chicken Rice a/311, Clementi Ave 2, #02-25 t/chickenRice t/opensDaily`<br>
+       Expected: A new stall is added to the list. Details of the newly added stall shown in the status message. GUI updates to show the newly added stall and its details.
+
+    3. Test case: `sadd n/Ah Kim Chicken Rice`<br>
+       Expected: No stall is added. Error details shown in the status message. GUI does not update.
+
+    4. Other incorrect stall add commands to try: `sadd`, `sadd a/311, Clementi Ave 2, #02-25`, `sadd n/Kim's Chicken a/311, Clementi Ave 2, #02-25` <br>
+        Expected: Similar to previous.
+
+### Adding a review
+
+1. Adding a review to a stall
+
+    1. Prerequisites: At least one stall entry exists
+
+    2. Test case: `radd s/1 d/20/09/2022 c/The food was good, the chicken rice was fresh. r/4 t/opensDaily t/worthyTrip`<br>
+       Expected: A new review is added to the list as the review of the first stall. Details of the newly added review shown in the status message. GUI updates to show the newly added review and its details.
+
+    3. Test case: `radd s/1 d/20/09/2022`<br>
+       Expected: No review is added. Error details shown in the status message. GUI does not update.
+
+    4. Other incorrect review add commands to try: `radd`, `radd s/1`, `radd s/1 c/The food was good`, `radd d/20/09/2022 c/The food was good, the chicken rice was fresh.` <br>
+        Expected: Similar to previous.
+
+### Listing all stalls
+
+1. Lists all stalls on GUI
+
+    1. Prerequisites: None
+   
+    2. Test case: `slist`<br>
+           Expected: GUI updates and all stalls are listed.
```

Comment:
This extra space seems to put this in a code block - do check. Perhaps make the spacing consistent to avoid this problem.

Reply:
Fixed.